include/seastar/util/print_safe.hh

```diff
@@ -61,6 +61,23 @@ void print_safe(const char *str) noexcept {
     print_safe(str, strlen(str));
 }
 
+// Fills a buffer with a hexadecimal representation of an integer
+// and returns a pointer to the first character.
+// For example, convert_hex_safe(buf, 4, uint16_t(12)) fills the buffer with "   c".
+template<typename Integral>
+SEASTAR_CONCEPT( requires std::is_integral_v<Integral> )
```

Comment:
Does it need `/sis_integral_v/integral/` as in the first patch  from this PR?

Reply:
no, it does not depend on the first patch in this PR. it is here just for preparing the 3rd commit in this PR.

Reply:
I meant -- in the first patch you did

```diff
template<typename Integral>
-SEASTAR_CONCEPT( requires std::is_integral_v<Integral> )
+SEASTAR_CONCEPT( requires std::integral<Integral> )
void convert_zero_padded_hex_safe(char *buf, size_t bufsz, Integral n) noexcept {
```

should this change be made here too?

Reply:
this is implemented in the 3rd commit. i wanted to make each commit more dedicated -- the goal of the 2nd one is just to move the function around.

Reply:
Ah, indeed. Missed that